browser/src/components/FoiaList.js

```diff
@@ -31,6 +36,26 @@ const FoiaList = (props) => { const [status, setStatus] = useState();
   filterData.prices = Array.from(uniquePrices).sort((a, b) => a - b);
   filterData.statuses = Array.from(uniqueStatuses);
 
+  const dt = DateTime.local();
```

Comment:
Currently, turnaround time is based on the current date, and will compare if the dt - turnaround time < the end_date.

Should this comparison be a compare of datetime_done and datetime_submitted?

Reply:
Yeah, that's interesting. We could certainly do it that way, however just to explain the rationale behind the current logic, it's that we're exposing how long things are still in process from the current moment, to highlight to journalists or activists requests that are really slow up to the moment. In this method we are discarding the done ones, since those have already been resolved.